libraries/rush-lib/src/logic/operations/IOperationExecutionResult.ts

```diff
@@ -31,6 +31,11 @@ export interface IOperationExecutionResult {
    * Object used to report a summary at the end of the Rush invocation.
    */
   readonly stdioSummarizer: StdioSummarizer;
+  /**
+   * The value indicates the duration of the same operation without cache hit.
+   * It is specified if current operation hits cache
+   */
+  readonly durationInSecondsWithoutCache: number | undefined;
```

Comment:
I'd call this `originalDurationInSeconds` or similar, to signify that it is the duration from the original run of an operation that was skipped/replayed